package.json

```diff
@@ -125,10 +125,19 @@
     "yargs": "6.6.0"
   },
   "dependencies": {
+    "@angular/common": "4.0.0-rc.2",
```

Comment:
ok soooo I've always thought `dependencies` are required to run, `devDependencies` are those things used for development.. since we run with da bundles it kinda makes most things `devDependencies`

http://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies#22004559